### DISCUSSION TITLE
Replace the title of the modal window with "Sort by..."

### DISCUSSION THREAD

**Discussion by irmod** [damionx7/Zettel-Notes-Documentation#170] [Feature Requests]
Hi

You might agree that if you replace the title of the modal window from "Select filter" to "**Sort by...**", it will be easier for the user to understand their actions. Three dots will make the title more human.

> The filtered list displays only rows that meet the criteria.  Unlike sorting, a filter does not change the order of entries in the list.  Filtering temporarily hides rows that you don't want to display.

Thank you for your attention

![Sort_by_02.png](https://github.com/user-attachments/assets/71fe49f7-0d7b-4d2a-9c5b-b28122025512)

**Comment by irmod** [comment]
@damionx7, perhaps you can find the time to make this small correction.
Swiss watches are valued for their attention to detail...

**Reply by damionx7** [reply]
Sure, I have updated the title.

**Comment by irmod** [comment]
Unexpected or strange behavior of the interface, as well as the working environment, is the reason for distrust of the product as a whole. Details matter! @damionx7, thank you for understanding me ❤️
